src/Spryk/Model/Spryk/Filter/ResourceNameToControllerNameFilter.php

```diff
@@ -48,6 +50,8 @@ public function filter(string $value): string
             $name = $parts[1];
         }
 
-        return ucfirst($name);
+        $filter = new DashToCamelCase();
```

Comment:
Can't you use the DI container for it? I would expect adding this as a dependency to the constructor auto injects it for you. Or is there any issue?

Reply:
I ran into a problem when injecting the `DashToCamelCase` through the constructor.

```
private DashToCamelCase $dashToCamelCase;

public function __construct(DashToCamelCase $dashToCamelCase)
{
    $this->dashToCamelCase = $dashToCamelCase;
}
```

Would result in this error:
```
In DefinitionErrorExceptionPass.php line 54:
                                                                               
  Cannot autowire service "SprykerSdk\Spryk\Model\Spryk\Filter\ResourceNameTo  
  ControllerNameFilter": argument "$dashToCamelCase" of method "__construct()  
  " references class "Laminas\Filter\Word\DashToCamelCase" but no such servic  
  e exists.                                                                    
                                                                               
...
```

I solved this by adding `$services->set(DashToCamelCase::class);` to `config/services.php`, but I'm unsure if this is the correct way to do this -- there's plenty of documentation on *how* to register configure/services, but not a lot is written about *when* to use services.

It felt out of place and I decided to keep the direct instantiation, since that is what is used for similar code in the project. Let me know if I should go ahead and add `DashToCamelCase` as a service like described above.

Reply:
IMO it would be best to make all filters available in one shot. 

```
$services->load('Laminas\\Filter\\', '../vendor/laminas/laminas-filter/');
```
If that doesn't work I'd use the approach you proposed.